ui/src/app/views/workflow/run/node/workflow.run.node.component.ts

```diff
@@ -90,6 +90,15 @@ export class WorkflowNodeRunComponent {
                 this.subNodeRun = this._workflowEventStore.selectedNodeRun().subscribe(wnr => {
                     this.nodeRun = wnr;
                     if (this.nodeRun) {
+                        if (PipelineStatus.isDone(wnr.status)) {
```

Comment:
Move the code here: https://github.com/ovh/cds/blob/master/ui/src/app/app.service.ts#L260
just before broadcasting the event.